exercises/pascals-triangle/src/example.c

```diff
@@ -0,0 +1,39 @@
+#include "pascals_triangle.h"
+
+void pt_free(size_t ** mem_block, size_t n)
+{
+   for (size_t i = 0; i < n; i++) {
+      free(mem_block[i]);
+   }
+   free(mem_block);
+}
+
+size_t **pascals_triangle(char *count)
+{
+   if (count == NULL) {
+      return (size_t **) NULL;
+   }
+   size_t **zero = (size_t **) calloc(1, sizeof(size_t *));
```

Comment:
Why do you allocate `zero` here if you only use it in the first `if` below? This forces you to deallocate when you don't use it which is actually in two different cases.